### ISSUE TITLE
Warning during bootstrap

### ISSUE THREAD

**Issue by tailhook** [edgedb/edgedb#3218]
- EdgeDB Version: 1.0-dev.6229+004227f

Warns during bootstrap:
```
Successfully installed 1.0-dev.6229+004227f
Initializing EdgeDB instance...
WARNING 268 2021-11-27T16:25:05.691 py.warnings: /home/user1/.local/share/edgedb/portable/1.0-dev.6229/lib/python3.10/site-packages/edb/server/server.py:1288: RuntimeWarning: coroutine '_signal_sysevent' was never awaited
  await binary.EdgeConnection.run_script(
```

**Comment by fantix** [comment]
should be fixed together with #3112

**Comment by tailhook** [comment]
Then #3112 should be prioritized I think. Seeing this warning each time you create an instance on 1.0 would be unfortunate. Or maybe we can come up with a simpler fix for just this warning?
